.github/actions/update-github-issue-and-pull-request/Update-GitHubIssueAndPullRequest.ps1

```diff
@@ -18,7 +18,13 @@ if ($Branch -NotLike '*issue*')
     Exit
 }
 
-$Branch -match '(\w+-\d+)'
+$Branch -match '(\w+-\d+)' | Out-Null
+
+if ($null -eq $matches)
```

Comment:
I came across this in the test dummy repos but is this actually a thing in the main repos? Because I have never seen the workflow fail due to the `$matches[0]` being `$null`

Reply:
No, but the branch names you used in the test repos triggered this error. The change here is to still allow the failure (because not having an issue key in the branch name after "issue/" is invalid), while also providing a descriptive error message at the actual point of the failure instead of throwing a seemingly unrelated error later.